### ISSUE TITLE
Join Us: Update alt text for Join Us - Header image to adhere to WCAG

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#3851]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, we need to provide clear and descriptive alt text on the `Join Us` page so that we adhere to the Web Content Accessibility Guidelines (WCAG) and we can achieve our mission of inclusivity.

### Action Items
- [x] Change the image alt property value within `pages/join-us.html`:

**From:**
```
alt="join us banner icon"
```

**To:**
```
alt=""
```

- [x] Ensure that the corresponding `Join Us webpage` stays the same after the change
- [x] Using developer tools to inspect the image, ensure that the new alt text gets incorporated after the change.
  - **Note:** `alt=""` in code while show up as `alt` when using developer tools to inspect the image's alt text property.

**For PM, Merge Team, or Tech Lead**
- [ ] Once the pull request associated with this issue is approved and merged, please update and edit epic #3858 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- Join Us webpage: https://www.hackforla.org/join

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `pages/join-us.html`
- See the source file code on the repo: https://github.com/hackforla/website/blob/gh-pages/pages/join-us.html
- This issue is part of epic #3858

#### Background information about the Jekyll framework (optional reading)
- [Jekyll Data Files](https://jekyllrb.com/docs/datafiles/)
- [Jekyll Front Matter](https://jekyllrb.com/docs/front-matter/)

#### Information about WCAG and alt text (optional reading)
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

**Comment by angieyan** [comment]
i. FEB 11-12
ii. 2 hours
